applicationinsights/TelemetryClient.py

```diff
@@ -200,7 +200,18 @@ def track_request(self, name, url, success, start_time=None, duration=None, resp
         data.id = str(uuid.uuid4())
         data.name = name
         data.start_time = start_time or datetime.datetime.utcnow().isoformat() + 'Z'
-        data.duration = duration or 0
+
```

Comment:
Note: we could use ```datetime.timedelta``` here, but it wouldn't really make the calculations any simpler.